LICENSE.txt

```diff
@@ -0,0 +1,214 @@
+LICENSE
+
+VMware vmw-guestinfo
+
+Copyright � 2017 VMware, Inc.  All rights reserved.
+
+This product is licensed to you under the Apache License version 2.0 (the "License").  You may not use this product except in compliance with the License.
+
+                                 Apache License

```

Comment:
https://github.com/vmware/vic/blob/master/LICENSE

Looks like that bit is in the `vic` License as well.  Interesting.  I got the standard apache license text ( http://apache.org/licenses/LICENSE-2.0.txt) and will use that.